rivershared/util/serviceutil/service_util.go

```diff
@@ -44,14 +44,23 @@ func CancellableSleepC(ctx context.Context, sleepDuration time.Duration) <-chan
 // into a ridiculously distant future. This constant is typically injected into
 // the CancellableSleepExponentialBackoff function. It could technically take
 // another value instead, but shouldn't unless there's a good reason to do so.
-const MaxAttemptsBeforeResetDefault = 10
+const MaxAttemptsBeforeResetDefault = 7
```

Comment:
Might make sense to call out the max sleep here so the choice of 7 is somewhat justified.